lib/plan_executor/app.rb

```diff
@@ -78,15 +69,24 @@ def validate_schema(schema, body)
       error = validate_schema(@schema, body)
       return [400, error.to_json] unless error.nil?
 
+      # Create a basic executor, leave concurrency up to Orchestrator.
+      # This is just a class variable for testing
+      executor = PlanExecutor::Executor.new(0, body['job_id'])
```

Comment:
The first argument to `PlanExecutor::Executor.new` appears to be `noop`, so `0` isn't an appropriate value for that. Also, this is a local variable, not a class variable.